test/spark-api.js

```diff
@@ -5,7 +5,11 @@ const recordTelemetry = (measurementName, fn) => { /* no-op */ }
 
 describe('spark-api client', () => {
   it('fetches round details', async () => {
-    const { retrievalTasks, ...details } = await fetchRoundDetails(520, recordTelemetry)
+    const { retrievalTasks, ...details } = await fetchRoundDetails(
+      '0x381b50e8062757c404dec2bfae4da1b495341af9',
+      10,
+      recordTelemetry
+    )
```

Comment:
This test is failing right now, it's waiting for deployment of https://github.com/filecoin-station/spark-api/pull/103

I'll also need to update the IE contract version & round index to make the test pass.

Reply:
Fixed in [6c3e5fb](https://github.com/filecoin-station/spark-evaluate/pull/32/commits/6c3e5fba71735949ba80ed2199d5c0ca1988ccbf)